docs/guides/getting-started.md

```diff
@@ -2,141 +2,75 @@
 
 <div class="vueschool"><a href="https://vueschool.io/lessons/installing-vue-test-utils?friend=vuejs" target="_blank" rel="sponsored noopener" title="Learn how to get started with Vue Test Utils, Jest, and testing Vue Components with Vue School">Learn how to get started with Vue Test Utils, Jest, and testing Vue Components</a></div>
 
-### Setup
+### What is Vue Test Utils?
 
-If you already have a project that was created with the [Vue CLI](https://cli.vuejs.org/), you might want to add and configure the [core Jest plugin](https://cli.vuejs.org/core-plugins/unit-jest.html) or the [core Mocha plugin](https://cli.vuejs.org/core-plugins/unit-mocha.html).
+Vue Test Utils (VTU) is a set of utility functions aimed to simplify testing Vue.js components. It provides some methods to **mount** and **interact** with Vue components in an isolated manner.
 
-If needed, check out the [Installation guides](../installation/README.md) for further details.
-
-### Mounting Components
-
-Vue Test Utils tests Vue components by mounting them in isolation, mocking the necessary inputs (props, injections and user events) and asserting the outputs (render result, emitted custom events).
-
-Mounted components are returned inside a [Wrapper](../api/wrapper/), which exposes many convenience methods for manipulating, traversing and querying the underlying Vue component instance.
-
-You can create wrappers using the `mount` method. Let's create a file called `test.js`:
+Let's see an example:
 
 ```js
-// test.js
-
-// Import the `mount()` method from the test utils
-// and the component you want to test
+// Import the `mount()` method from Vue Test Utils
 import { mount } from '@vue/test-utils'
-import Counter from './counter'
-
-// Now mount the component and you have the wrapper
-const wrapper = mount(Counter)
-
-// You can access the actual Vue instance via `wrapper.vm`
-const vm = wrapper.vm
-
-// To inspect the wrapper deeper just log it to the console
-// and your adventure with the Vue Test Utils begins
-console.log(wrapper)
-```
-
-### Test rendered HTML output of the component
-
-Now that we have the wrapper, the first thing we can do is to verify that the rendered HTML output of the component matches what is expected.
 
-```js
-import { mount } from '@vue/test-utils'
-import Counter from './counter'
-
-describe('Counter', () => {
-  // Now mount the component and you have the wrapper
-  const wrapper = mount(Counter)
-
-  it('renders the correct markup', () => {
-    expect(wrapper.html()).toContain('<span class="count">0</span>')
+// The component to test
+const MessageComponent = {
+  template: '<p>{{ msg }}</p>',
+  props: ['msg']
+}
+
+test('displays message', () => {
+  // mount() returns a wrapped Vue component we can interact with
+  const wrapper = mount(MessageComponent, {
+    props: {
```

Comment:
I think this is `propsData` in VTU V1 🤔 

We should backport `props` perhaps.

Reply:
dang, you're right. fixed.

and yeah, maybe porting back `props` to match VTU 2 is a sensible idea 🤔